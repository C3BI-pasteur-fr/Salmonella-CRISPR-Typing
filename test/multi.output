>squery1_Aba16
gatcgatcgatgctacgtagctagctagctagctagtcgatgcatcacatcatcg-Aba1
6|&tacgatcgtacgtactgatcgtagctagctgactgatcgatcgtcacgatcggcatc
atcgt
>query2_NiaB3
tagctagctagctagctgatcgatcgatgctagcatgcatcgtagctacgtacgtacgta
cgtagctacgtagctacgtagctagctagctagctagctacgtacgtagctacgtagcta
cgtacgta-NiaB3|&actgatgctagcgatcgatgctaatcgctagctagctagcagct
agctagtcgatcaggcagtcgatgcag
>query3_SalB69_BraB6
atcgatcgatgcagctacgtagctagctagctgatcgatcgagcagctctcgttaaatca
ttagcggact-SalB69|&cgatcgatcgaggcatgcgtatgcgagtctgagcttagctg
tgtagcactgatcatctatcgagcgatgctaggtcagcgtagtcaggtactg-BraB6|&
tacgtactgattatctagtctagtgcacgagcgtatcagcgagctagcatgcgatgctga
cgacgtagctgatgcatgctagacgagct
