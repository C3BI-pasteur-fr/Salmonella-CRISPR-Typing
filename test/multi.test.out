>query1 <unknown description>
aa-spacer1&tcccggg
>query2 <unknown description>
aa-spacer2&tcccggg
